File: src/Estudiante.cpp
#include "Estudiante.h"
#include <ctype.h>

Estudiante::Estudiante(string carnet_,string dpi_, string name_, string carrera_, string email_, string pass_, int credit_, int edad_)
{
    carnet = carnet_;
    dpi = dpi_;
    name = name_;
    carrera = carrera_;
    email = email_;
    pass = pass_;
    credit = credit_;
    edad = edad_;

}
//setters
void Estudiante::setcarnet(string carnet_){
    this->carnet =carnet_;
}

void Estudiante::setdpi(string dpi_){
    this->dpi =dpi_;
}

void Estudiante::setname(string name_){
    this->name = name_;
}

void Estudiante::setcarrera(string carrera_){
    this->carrera = carrera_;
}

void Estudiante::setemail(string email_){
    this->email = email_;
}

void Estudiante::setcredit(int credit_){
    this->credit = credit_;
}
void Estudiante::setedad(int edad_){
    this->edad = edad_;
}

//getters
string Estudiante::getcarnet(){
    return this->carnet;
}

string Estudiante::getdpi()
{
    return this->dpi;
}

string Estudiante::getname()
{
    return this->name;
}

string Estudiante::getcarrera(){
    return this->carrera;
}

string Estudiante::getemail(){
    return this->email;
}

string Estudiante::getpass(){
    return this->pass;
}

int Estudiante::getcredit(){
    return this->credit;
}

int Estudiante::getedad(){
    return this->edad;
}

File: Estructuras/estudiante.py
class estudiante:
    def __init__(self, ncarnet_, dpi_, nombre_, carrera_, correo_, passw_, credit_, edad_, yearlist_):
        self.carnet = ncarnet_
        self.dpi = dpi_
        self.nombre = nombre_
        self.carrera = carrera_
        self.correo = correo_
        self.passw = passw_
        self.credito = credit_
        self.edad = edad_
        self.yearlist = yearlist_


    def getcarnet(self):
        return  self.carnet

    def getdpi(self):
        return self.dpi

    def getnombre(self):
        return self.nombre

    def getcarrera(self):
        return  self.carrera

    def getcorreo(self):
        return self.correo

    def getpassw(self):
        return self.passw

    def getcredito(self):
        return self.credito

    def getedad(self):
        return self.edad

